src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2081,6 +2087,48 @@ private bool RefMustBeObeyed(bool isDelegateCreation, ArgumentSyntax argumentSyn
             return hadError;
         }
 
+        private BoundExpression BindArgumentValue(DiagnosticBag diagnostics, ArgumentSyntax argumentSyntax, bool allowArglist, RefKind refKind)
+        {
+            if (argumentSyntax.Expression != null)
+            {
+                return BindArgumentExpression(diagnostics, argumentSyntax.Expression, refKind, allowArglist);
+            }
+
+            var typeSyntax = argumentSyntax.Type;
+
+            bool isConst = false;
+            bool isVar;
+            AliasSymbol alias;
```

Comment:
Thanks, I will make sure to add this test in the next PR.

---
In reply to: [63742937](https://github.com/dotnet/roslyn/pull/11393#discussion_r63742937) [](ancestors = 63742937)